python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -945,6 +946,66 @@ def create_array(dtype):
         dtype=dtype)
 
 
+def equal(x, y, cond=None, **ignored):
+    """
+    **equal**
+
+    This layer returns the truth value of :math:`x == y` elementwise.
+
+    Args:
+        x(Variable): First operand of *equal*
+        y(Variable): Second operand of *equal*
+        cond(Variable|None): Optional output variable to store the result of *equal*
+
+    Returns:
+        Variable: The tensor variable storing the output of *equal*.
+
+    Examples:
+        .. code-block:: python
+
+          less = fluid.layers.equal(x=label, y=limit)
+    """
+    helper = LayerHelper("equal", **locals())
+    if cond is None:
+        cond = helper.create_tmp_variable(dtype='bool')
+        cond.stop_gradient = True
+
+    helper.append_op(
+        type='equal', inputs={'X': [x],
+                              'Y': [y]}, outputs={'Out': [cond]})
+    return cond
+
+
+def less_equal(x, y, cond=None, **ignored):
```

Comment:
The comment does not need to write by hand.

You can import `from layer_function_generator import autodoc`, and use `autodoc()` to load C++ comments to Python.

The example is [here](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/layers/device.py#L25)

Reply:
If you want to add more comments in Python, just use

```
@autodoc(comment="""example:
   ....
""")
```

Reply:
@reyoung  It's a cool feature. But the  `Args` in python API is different with these in C++ comments. And a formated example python code is expected. So It is more convenient  rewritting the doc in Python API.